src/api/Folder.js

```diff
@@ -305,7 +320,7 @@ class Folder extends Item {
         }
 
         // Make the XHR request
-        this.folderRequest();
+        this.folderRequest(options);
```

Comment:
Looks like this is no longer needed?

Reply:
This actually is still needed in the current version of the code, since `ContentExplorer` passes `fields` into `options`.  (Previously, `fields` was not defined in `options` and defaulted to `FOLDER_FIELDS_TO_FETCH`, but now, since `ContentExplorer` determines the fields based on `features`, I think this is needed.)